frontend/src/App/App.jsx

```diff
@@ -1,6 +1,10 @@
 import React from 'react'
 import Landing from '../Landing/Landing'
 import Login from '../Login/Login'
+import MyProfile from '../MyProfile/MyProfile'
+import Patients from '../Patients/Patients'
+import Locate from '../Locate/Locate'
+import Covid from '../Covid/Covid'
```

Comment:
i think i might have given this feedback on your plan, but did we want to keep these as separate pages or combine location and COVID together as info?
also, let's keep these as stretch features for now, we can revisit once your TCs are complete!